bitcoind-rpc/src/Bitcoin/Core/RPC/Transactions.hs

```diff
@@ -247,7 +248,7 @@ instance ToJSON PsbtInput where
 
 -- | @since 0.3.0.0
 data PsbtOutputs = PsbtOutputs
-    { psbtOutputAddrs :: [(Text, Word64)]
+    { psbtOutputAddrs :: [(Key, Word64)]
```

Comment:
This really should be `Text`.  If necessary, we should adapt the `ToJSON` instance.